inc/scss-compiler.php

```diff
@@ -212,10 +212,19 @@ function bootscore_child_has_scss() {
 }
 
 function bootscore_compile_scss() {
-  $scss_compiler = new BootscoreScssCompiler();
-  $scss_compiler->scssFile('/assets/scss/main.scss')
-                ->cssFile('/assets/css/main.css')
-                ->addModifiedCheckTheme()
-                ->addModifiedCheck(get_template_directory() . '/assets/scss/bootstrap/bootstrap.scss', false)
-                ->compile();
+  // Compile the main.scss file
+  $scss_compiler_main = new BootscoreScssCompiler();
+  $scss_compiler_main ->scssFile('/assets/scss/main.scss')
+                      ->cssFile('/assets/css/main.css')
+                      ->addModifiedCheckTheme()
+                      ->addModifiedCheck(get_template_directory() . '/assets/scss/bootstrap/bootstrap.scss', false)
+                      ->compile();
+  
+  // Compile the editor.scss file
+  $scss_compiler_editor = new BootscoreScssCompiler();
+  $scss_compiler_editor ->scssFile('/assets/scss/editor.scss')
+                        ->cssFile('/assets/css/editor.css')
+                        ->addModifiedCheckTheme()
+                        ->addModifiedCheck(get_stylesheet_directory() . '/assets/scss/main.css', false)
+                        ->compile();
```

Comment:
I wouldn't do it like this. This means every time main is compiled (which is each time in dev mode), it also compiles the editor.css. I would recommend only checking the bootstrap files and the variables. It may also be smart to add an additional function to the compiler to skip the env check.

Reply:
Agree, want to change this quickly? I‘m not in front of my machine.

Reply:
Already got it done 😉

Reply:
Perfect, merge?